.gitignore

```diff
@@ -1,4 +1,5 @@
 /scylla-operator
+/scylla-operator-tests
```

Comment:
20MB Binary is commited as a part of "api: Move v1 api into group folder", and later this ignore is added which doesn't have affect on previous commits.

Reply:
Oh, must have got there during the hundreds of sync commits when I was fighting GH actions to get it working. Thanks for noticing.